README.md

```diff
@@ -131,6 +130,11 @@ Usage:
             The given day at 7:00 am
             e.g. 22.12.2020
             e.g. 01.01.21
+
+        {start_value}/{end_value}/{time_delta}
+            Calculate the finish time according specified start/end values.
+            e.g. 0/100/1m -> 100 minutes
+            e.g. 15/35/17s -> 340 seconds
```

Comment:
Could you explain when we should use this option? I don't understand what start/end values are and why they're used to calculate `multiplier`.

Reply:
Sometimes I want to be reminded when some tasks are done. Suppose a task has 35 steps and currently it is on the 15th step, and each step takes 5 minutes. Then I will create a reminder in `(35 - 15) * 5` minutes. With this option, I don't have to calculate the number manually and can specify `15/35/5m` as the reminder.